Target-MVD-Mathias/Networking/Services/UserAPI.swift

```diff
@@ -74,6 +74,27 @@ class UserAPI {
       failure(error)
     }
   }
+  
+  class func signup(name: String, email: String, password: String, gender: String, success: @escaping (_ responseObject: [String: Any]) -> Void, failure: @escaping (_ error: Error) -> Void) {
+    let parameters = [
```

Comment:
Hi! We use 2 spaces for identation, you can see our style guide [here](https://github.com/toptier/guides/tree/master/swift). 
You can change the default value going to Xcode-> Preferences->Text editing-> Identation.